app/admin/councillor_contributions.rb

```diff
@@ -14,6 +14,8 @@
       row :contributor
       row :id
       row :authority
+      row :source_url
```

Comment:
The source information might not actually be a URL, so I recon we should just call this attribute "source".